ambari-server/src/main/java/org/apache/ambari/server/state/fsm/event/Event.java

```diff
@@ -26,5 +26,6 @@
 
   TYPE getType();
   long getTimestamp();
+  @Override
```

Comment:
why would we override method declared in the interface?  Well, i have no clue if this method declaration makes any sense at all - coz it is present in the extended interface

Reply:
OK, I'll remove it.